sbb_newspapers_org_image/eynollah.py

```diff
@@ -1,306 +1,198 @@
-        if len(np.shape(seg)) == 3:
-            seg = seg[:, :, 0]
-
-        seg_img = np.zeros((np.shape(seg)[0], np.shape(seg)[1], 3)).astype(float)
-        # colors=sns.color_palette("hls", n_classes)
-        colors = [[1, 0, 0], [8, 0, 0], [2, 0, 0], [4, 0, 0]]
-
-        for c in ann_u:
-            c = int(c)
-            segl = (seg == c)
-            seg_img[:, :, 0][seg == c] = colors[c][0]  # segl*(colors[c][0])
-            seg_img[:, :, 1][seg == c] = colors[c][1]  # seg_img[:,:,1]=segl*(colors[c][1])
-            seg_img[:, :, 2][seg == c] = colors[c][2]  # seg_img[:,:,2]=segl*(colors[c][2])
-        return seg_img
-
-    def rotate_image(self, img_patch, slope):
-        (h, w) = img_patch.shape[:2]
-        center = (w // 2, h // 2)
-        M = cv2.getRotationMatrix2D(center, slope, 1.0)
-        return cv2.warpAffine(img_patch, M, (w, h), flags=cv2.INTER_CUBIC, borderMode=cv2.BORDER_REPLICATE)
-    
-    def rotyate_image_different(self,img,slope):
-        #img = cv2.imread('images/input.jpg')
-        num_rows, num_cols = img.shape[:2]
-
-        rotation_matrix = cv2.getRotationMatrix2D((num_cols/2, num_rows/2), slope, 1)
-        img_rotation = cv2.warpAffine(img, rotation_matrix, (num_cols, num_rows) ) 
-        return img_rotation
-
-    def cleaning_probs(self, probs: np.ndarray, sigma: float) -> np.ndarray:
-        # Smooth
-        if sigma > 0.:
-            return cv2.GaussianBlur(probs, (int(3 * sigma) * 2 + 1, int(3 * sigma) * 2 + 1), sigma)
-        elif sigma == 0.:
-            return cv2.fastNlMeansDenoising((probs * 255).astype(np.uint8), h=20) / 255
-        else:  # Negative sigma, do not do anything
-            return probs
-
-    def crop_image_inside_box(self, box, img_org_copy):
-        image_box = img_org_copy[box[1]:box[1] + box[3], box[0]:box[0] + box[2]]
-        return image_box, [box[1], box[1] + box[3], box[0], box[0] + box[2]]
-
-    def otsu_copy(self, img):
-        img_r = np.zeros(img.shape)
-        img1 = img[:, :, 0]
-        img2 = img[:, :, 1]
-        img3 = img[:, :, 2]
-        # print(img.min())
-        # print(img[:,:,0].min())
-        # blur = cv2.GaussianBlur(img,(5,5))
-        # ret3,th3 = cv2.threshold(blur,0,255,cv2.THRESH_BINARY+cv2.THRESH_OTSU)
-        retval1, threshold1 = cv2.threshold(img1, 0, 255, cv2.THRESH_BINARY + cv2.THRESH_OTSU)
-        retval2, threshold2 = cv2.threshold(img2, 0, 255, cv2.THRESH_BINARY + cv2.THRESH_OTSU)
-        retval3, threshold3 = cv2.threshold(img3, 0, 255, cv2.THRESH_BINARY + cv2.THRESH_OTSU)
-
-        img_r[:, :, 0] = threshold1
-        img_r[:, :, 1] = threshold1
-        img_r[:, :, 2] = threshold1
-        return img_r
-
-    def otsu_copy_binary(self,img):
-        img_r=np.zeros((img.shape[0],img.shape[1],3))
-        img1=img[:,:,0]
-
-        retval1, threshold1 = cv2.threshold(img1, 0, 255, cv2.THRESH_BINARY+cv2.THRESH_OTSU)
 
-        
+        self.model_dir_of_enhancemnet = dir_models + "/model_enhancement.h5"
```

Comment:
`enhancemnet` /  `enhancement` (check for usages)

Reply:
There are many more such small typos, like `seperate` or `rotyate`, `teilwiese` etc. I suggest a separate PR only for fixing typos and renaming variables/functions to be more expressive.

Reply:
Yes, we should make a separate PR for all these and perhaps the package renaming after this one is merged.